addons/account/models/account.py

```diff
@@ -705,7 +705,8 @@ def compute_all(self, price_unit, currency=None, quantity=1.0, product=None, par
                 taxes += ret['taxes']
                 continue
 
-            tax_amount = tax._compute_amount(base, price_unit, quantity, product, partner)
+            tax_excluded = tax_included = tax_base = base
```

Comment:
I agree about `tax_base`, the others don't seem necessary. OK, it doesn't hurt to compute these values, but if we don't use them in the end, I would avoid 'noise' in the code.